source/deploy/server/server-deployment-planning.rst

```diff
@@ -10,6 +10,7 @@ Server deployment planning
     Deploy with Kubernetes </deploy/server/deploy-kubernetes>
     Deploy with Linux </deploy/server/deploy-linux>
     Deploy with Containers </deploy/server/deploy-containers>
+    Deploy for Out-of-Band Use </deploy/server/deploy-oob>
```

Comment:
@stafot - Any changes you'd like to see made to the navigation option for this new page?